packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -338,14 +336,11 @@ impl VTag {
         match &self.inner {
             VTagInner::Other { children, .. } => children,
             _ => {
-                static mut EMPTY: MaybeUninit<VList> = MaybeUninit::uninit();
-                static ONCE: Once = Once::new();
-                unsafe {
-                    ONCE.call_once(|| {
-                        EMPTY = MaybeUninit::new(VList::default());
-                    });
-                    &*EMPTY.as_ptr()
-                }
+                // This is mutable because the VList is not Sync
+                static mut EMPTY: VList = VList::new();
```

Comment:
```suggestion
                const EMPTY: VList = VList::new();
```
If we don't mutate it, there's no need to make it `mut`. We can also make it `const` instead of `static` so why not do it?

Reply:
For const value inside a function, rustc produce this error:
```
error[E0515]: cannot return reference to temporary value
   --> packages/yew/src/virtual_dom/vtag.rs:343:17
    |
343 |                 &EMPTY
    |                 ^-----
    |                 ||
    |                 |temporary value created here
    |                 returns a reference to data owned by the current function
```

Reply:
> If we don't mutate it, there's no need to make it `mut`.

I wrote:
```
// This is mutable because the VList is not Sync
```
Immutable `static EMPTY: VList = VList::new()` does not compile.

Reply:
Welp, seems like have no other choice. I thought maybe `const` would work but it doesn't because lifetimes.